yarl/_helpers_c.pyi

```diff
@@ -0,0 +1,6 @@
+from typing import Any
+
+class cached_property:
+    def __init__(self, wrapped: Any) -> None: ...  # pragma: no cover
+    def __get__(self, inst: Any, owner: Any) -> Any: ...  # pragma: no cover
+    def __set__(self, inst: Any, value: Any) -> None: ...  # pragma: no cover
```

Comment:
```suggestion
    def __init__(self, wrapped: Any) -> None: ...
    def __get__(self, inst: Any, owner: Any) -> Any: ...
    def __set__(self, inst: Any, value: Any) -> None: ...
```

This looks like the same problem with multi-line test coverage with the reporting and not a real issue